admin-dev/themes/default/template/helpers/list/list_header.tpl

```diff
@@ -352,6 +352,37 @@
 											});
 										</script>
 									</div>
+                {elseif $params.type == 'datetime'}
+                  <div class="date_range row">
+                    <div class="input-group fixed-width-md center">
+                      <input type="text" class="filter datetimepicker date-input form-control" id="local_{$params.id_date}_0" name="local_{$params.name_date}[0]"  placeholder="{l s='From'}" />
+                      <input type="hidden" id="{$params.id_date}_0" name="{$params.name_date}[0]" value="{if isset($params.value.0)}{$params.value.0}{/if}">
+                      <span class="input-group-addon">
+                    <i class="icon-calendar"></i>
```

Comment:
indentation is really weird, can you improve that before review?

Reply:
Thanks! I've changed it. :)